### ISSUE TITLE
Scrub .travis.yml file

### ISSUE THREAD

**Issue by david-a-wheeler** [metamath/set.mm#1120]
@benjub looked at the .travis.yml file and noted a number of things that probably need fixing:

~~~~
l.48--50: now obsolete ?
l.58: change to " # Create mmset.html and mmil.html so we can verify against them." ?
l.62: why are the options different from l.61 ?
l.67: why no analogue for some "idiscouraged" ?
l.68: comment is actually describing l.65,66 ?
l.69: now redundant with l.66 ?
l.78-84: this generates mpe but not ile ?
~~~~

**Comment by david-a-wheeler** [comment]
This was originally raised in #1118 .

@jkingdon is working on a follow-up pull request to make some fixes to .travis.yml.  Once that's merged, we should look at the resulting file, at least to examine the questions raised above.

**Comment by jkingdon** [comment]
> @jkingdon is working on a follow-up pull request to make some fixes to .travis.yml. Once that's merged, we should look at the resulting file, at least to examine the questions raised above

I assume you are thinking of #1121 . I don't have anything planned beyond that, so no need to wait on me for any of these things.

**Comment by david-a-wheeler** [comment]
> l.48--50: now obsolete ?

Correct, I'll remove them.

l.58: change to " # Create mmset.html and mmil.html so we can verify against them." ?
> l.62: why are the options different from l.61 ?

We have:

~~~~
  - scripts/verify --top_date_skip --extra 'write bibliography mmbiblio.html' set.mm
  - scripts/verify iset.mm
~~~~

The "top_date_skip" is important when there are many simultaneous contributors; we don't yet have that many.  Writing the bibilography is plausible, but I'll let @jkingdon propose that.

> l.67: why no analogue for some "idiscouraged" ?

There really should be.  I'll name it "iset-discouraged" to be consistent with the existing script.

> l.68: comment is actually describing l.65,66 ?

I'll fix up the comment.

> l.69: now redundant with l.66 ?

Correct. Removing.

> l.78-84: this generates mpe but not ile ?

No, though I understand the confusion. The regen_from_raw script does indeed regenerate ile, but this is only a test, and I move them into subdirectories so I can look at them later for debugging.  In actuality the *real* files are generated at a separate time.
